src/rich-log/domain.hh

```diff
@@ -0,0 +1,131 @@
+#pragma once
+
+#include <clean-core/macros.hh>
+
+namespace rlog
+{
+namespace verbosity
+{
+enum type : int
+{
+    /// TRACE is used to provide diagnostic information verbosely and in very small steps
+    /// e.g. logging in each iteration of an algorithm
+    /// per default, trace should be compile-time DISABLED (as it can and will slow down the app)
```

Comment:
Agree on this idea but i would clarify the text:
"Trace is compile-time disabled on the Default domain"

Reply:
reorganized